build-phar.sh

```diff
@@ -1,14 +1,17 @@
 #!/usr/bin/env bash
 
-if [ ! -f box.phar ]; then
-    wget https://github.com/humbug/box/releases/download/3.0.0-beta.4/box.phar -O box.phar
-fi
+set -e
 
-# lock PHP to minimum allowed version
-composer config platform.php 7.2.0
-composer update --no-dev
+mkdir -p tmp
```

Comment:
Probably better to not work in the CWD

Reply:
What if we used a docker image with box to generate the phar? Something like https://github.com/jakzal/phpqa which comes with box installed

Reply:
I'd generally prefer to have the latest version of box, not a tag from a derived docker image.

I think the patch is fine, but CWD needs to be `/tmp/something` rather than `./tmp` 👍

Reply:
If that's OK with Travis CI (writing to /tmp) I can make the change. Some CIs don't let you go outside the working dir.

Reply:
Yes, travis is pretty much allowing everything